lib/matplotlib/backends/backend_svg.py

```diff
@@ -500,11 +500,12 @@ def _get_hatch(self, gc, rgbFace):
         edge = gc.get_hatch_color()
         if edge is not None:
             edge = tuple(edge)
-        dictkey = (gc.get_hatch(), rgbFace, edge)
+        lw = gc._linewidth
```

Comment:
Should be hatch, not regular, line width:
```suggestion
        lw = gc.get_hatch_linewidth()
```
which should fix the tests.